portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,102 @@
 // Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-/**
- * Adds a random greeting to the page.
- */
+
+let elementsToShow;
+let scroll;
+
+$(document).ready(function() {
+    scroll = window.requestAnimationFrame || function(callback){ window.setTimeout(callback, 1000/60)};
+    elementsToShow = document.querySelectorAll('.animate');    
+    loop();
+    asyncRemovePlaceHolder();
+})
+
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    revertToUnchecked();
+
+    const greetings =
+        ["\"In a world too often governed by corruption and arrogance, it can be difficult to stay true to one's philosophical and literary principles.\"   -Lemony Snicket, A Series of Unfortunate Events", "\"Did you know that the Sparrow flies south for winter?\"   -Skulduggery Pleasant, Skulduggery Pleasant", "\"Someday in your life, you'll realize there is no turning back. There was never any turning back.\"   -u/JayStar1213"];
+    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  
+    let entrances = 
+        ["animate__bounceInUp", "animate__bounceInLeft", "animate__bounceInRight", "animate__bounceInDown", "animate__fadeInBottomLeft", "animate__fadeInBottomRight"]
+    let entrance = entrances[Math.floor(Math.random() * greetings.length)];
+
+    const quote = document.getElementById('quote');
+    quote.className="";
+    quote.classList.add('italic');
+    quote.classList.add('animate__animated');
+    quote.classList.add(entrance);
+    quote.classList.add('animate__delay-1s');
```

Comment:
why are you setting the classes here, instead of defining them in html file.

Reply:
I'm setting the classes here because I want the quote div to have a different css animation each time a user presses the "click me" button to get the quote. Therefore I have to clear the classes of the quote, and then assign them new ones. 

Is there an easier way to do it? I'm also not sure what you mean by "instead of defining them in html file", could you explain more on that please? Thanks!

Reply:
i see. i don't know of a simpler way of doing what you want.
my advice would be to reconsider whether you want to support this feature. tbh, i didn't notice that there were different animations when i was using the site.

Reply:
I noticed the bug too! I fixed it by completely removing the div and creating it again in javascript.